lib/kafo/kafo_configure.rb

```diff
@@ -338,7 +338,15 @@ def set_app_options
                  :default => false, :advanced => true
       app_option ['-v', '--[no-]verbose'], :flag, 'Display log on STDOUT instead of progressbar',
                  :default => config.app[:verbose]
-      app_option ['-l', '--verbose-log-level'], 'LEVEL', 'Log level for verbose mode output',
+      log_levels = <<~HEREDOC.chomp
+        Log level for log based terminal output. The available levels are
+        ERROR  - Only show errors which prevented the installer from completing successfully.
+        WARN   - Deprecation warnings and other information users may want to be aware of.
+        NOTICE - High level information about installer execution and progress.
```

Comment:
I think this is incorrect. https://en.wikipedia.org/wiki/Syslog#Severity_level has a nice table and describes notice as:
> Normal but significant conditions. Conditions that are not error conditions, but that may require special handling.

Reply:
The same resource also states, "the meaning of severity levels other than Emergency and Debug are relative to the application" which I would argue is applicable in this case. The user's first impression of [NOTICE] level will now be the terminal output they see when running the installer, which typically doesn't suggest or require much special handling. "Normal but significant conditions" is perhaps more in line with how we are using the level.... the description I proposed here is like a version of "normal but significant conditions" but worded specifically for the installer.